kakao/src/main/kotlin/com/agoda/kakao/common/views/KBaseView.kt

```diff
@@ -76,6 +79,17 @@ open class KBaseView<out T> : BaseActions, BaseAssertions {
         function(this as T)
     }
 
+    fun configure(configuratorBuilderAction: ConfiguratorBuilder.() -> Unit) {
+        val configuratorBuilder = ConfiguratorBuilder.createWithHistory(KakaoConfigurator.configurator)
+        configuratorBuilderAction.invoke(configuratorBuilder)
```

Comment:
`ConfiguratorBuilder.createWithHistory(...).apply(configuratorBuilderAction)`
Also, naming here is too excessive, can be a lot shorter.

Reply:
I simplified it by putting 'history' argument into the constructor of Builder.